test-browser/utils/dataAttributes.spec.ts

```diff
@@ -1,27 +1,12 @@
-import { DataAttribute, getDataAttributesFromEventPath } from '../../src/utils/dataAttributes';
 import { createHtmlFixture } from '../test-helpers/createHtmlFixture';
-import { createTestEventManager } from '../test-helpers/createTestEventManager';
+import { parseEditButtonDataAttributes, parsePlusButtonDataAttributes } from '../../src/utils/dataAttributes';
+import { InsertPosition } from '../../src/lib/IFrameCommunicatorTypes';
 
 describe('dataAttributes.ts', () => {
-  const windowTestEventManager = createTestEventManager(window);
   const fixture = createHtmlFixture();
 
-  describe('getDataAttributesFromEventPath', () => {
-    function testClickEventParsedAttributes(clickTarget: HTMLElement, expected: Map<string, string>, done: DoneFn) {
-      windowTestEventManager.addEventListenerForCurrentTest(
-        'click',
-        (event) => {
-          const parsed = getDataAttributesFromEventPath(event);
-          expect(parsed).toEqual(expected);
-          done();
-        },
-        true
-      );
-
-      clickTarget.click();
-    }
-
-    it('should parse data attribute from event path', (done) => {
+  describe('parseEditButtonDataAttributes', () => {
```

Comment:
It seems that all unit tests check optimistic scenarios. Would it make sense to add some more, that will make sure that also some invalid cases will be handled in the way we want? For example, not found target element, missing element codename, etc.

Reply:
Yes, we can create more tests in the future.